x-pack/plugins/secops/public/containers/who_am_i/who_am_i.gql_query.ts

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
I think the kibana convention is all lowercase 🐍 case, so this would be `who_am_i.gql_query.ts`

Reply:
It is!!!